spec/features/admin/twitter/admin_sources_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Admin Twitter source new' do
+  context 'user with admin privileges' do
+    scenario 'can create new source for his organization' do
```

Comment:
most likely the names are set wrongly for these test cases - should this one be below?